packages/modules/mongodb/src/mongodb-container.ts

```diff
@@ -1,68 +1,85 @@
-import { AbstractStartedContainer, ExecResult, GenericContainer, StartedTestContainer, Wait } from "testcontainers";
+import { satisfies } from "compare-versions";
+import { AbstractStartedContainer, GenericContainer, StartedTestContainer, Wait } from "testcontainers";
 
 const MONGODB_PORT = 27017;
 
 export class MongoDBContainer extends GenericContainer {
+  private username = "";
+  private password = "";
```

Comment:
Let's remove the default values and have the type `string | undefined`, so I don't have to double check `if (this.username && this.password)`  works as expected